SMS/views.py

```diff
@@ -0,0 +1,68 @@
+from django.shortcuts import render
+from django.views.decorators.csrf import csrf_exempt
+from django.http import HttpResponse, HttpResponseBadRequest, JsonResponse
+from django.views.decorators.http import require_GET, require_POST
+
+from models import SMS
+
+from common import util, commonHttp
+import logging
+import json
+import requests
+
+expectedAttr = {
+	'TO': "to",
+	'TITLE': "title",
+	'MESSAGE': "message",
+}
+
+logger = logging.getLogger("django")
+
+@require_POST
+@csrf_exempt
+def create(request):
+	try:
+		json_obj = commonHttp.get_json(request.body)
+
+		req_attrs = [
+			expectedAttr["TO"], 
+			expectedAttr["TITLE"], 
+			expectedAttr["MESSAGE"]
+			]
+
+		commonHttp.check_keys(json_obj, req_attrs)
+
+		new_sms = SMS(
+			to=json_obj[expectedAttr["TO"]],
+			title=json_obj[expectedAttr["TITLE"]],
+			message=json_obj[expectedAttr["MESSAGE"]]
+			)
+
+
+
+		commonHttp.save_model_obj(new_sms)
+
+		url = "http://smsgateway.me/api/v3/messages/send"
+		data = {'email':'xiaojia1993@gmail.com', 'password':'','device':'31268','number': json_obj[expectedAttr["TO"]],'message': json_obj[expectedAttr["TITLE"]] + "," +json_obj[expectedAttr["MESSAGE"]]}
+
+		r=requests.post(url, data)
+		print "test"
```

Comment:
Ok nevermind, since I accidentally merged it in. Could @jiaqi1993 settle this locally?